packages/cli/commons/src/preconditions/apiKeyPrivilege.ts

```diff
@@ -34,7 +33,7 @@ export function HasNecessaryCoveoPrivileges(
             Boolean(anonymous)
           );
           throw new PreconditionError(message, {
-            category: PreconditionErrorCategory.MissingPlatformPrivilege,
+            category: PRECONDITION_ERROR_CATEGORY,
```

Comment:
Why redefining the variable?

Reply:
Because the precondition error might be defined in their own packages, the enum doesn't work anymore.
So I used const instead

Reply:
Ok, in that case, I would just get rid of the enum and only accept a string